src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
When I was testing by extending the manager classes on a sample app, I used those extension methods when I overrode some methods. Since the app developer uses them in custom classes, I chose not to move it to internal namespace